### PR TITLE
Set header guards to pragma once

### PR DESCRIPTION
**Pull request by tr29038**
I believe that is all of the #ifndef #define #endif header guards replaced

### CODE REVIEW

File: be/src/formats/orc/apache-orc/c++/src/io/InputStream.hh [hunk 1]
```diff
@@ -20,8 +20,7 @@
  * limitations under the License.
  */
 
-#ifndef ORC_INPUTSTREAM_HH
-#define ORC_INPUTSTREAM_HH
```
Comment (imay) [inline be/src/formats/orc/apache-orc/c++/src/io/InputStream.hh:24]: Corresponding `#endif` need to be deleted

Comment (tr29038) [reply be/src/formats/orc/apache-orc/c++/src/io/InputStream.hh:24]: Removed it!

File: be/src/gutil/linux_syscall_support.h [hunk 2]
```diff
@@ -127,8 +127,7 @@
  *      sys_stat64
  *      sys_write
  */
-#ifndef SYS_LINUX_SYSCALL_SUPPORT_H
-#define SYS_LINUX_SYSCALL_SUPPORT_H
```
Comment (imay) [inline be/src/gutil/linux_syscall_support.h:131]: This `endif` need to be removed.

Comment (tr29038) [reply be/src/gutil/linux_syscall_support.h:131]: There was already a missing endif in that file.  I just re-checked and I still do not see a corresponding endif.

Comment (imay) [reply be/src/gutil/linux_syscall_support.h:131]: If you delete this header guard, the last `#endif` int this file should be deleted.
Or you can skip this file and keep it as before.

Comment (tr29038) [reply be/src/gutil/linux_syscall_support.h:131]: I have deleted it.

**Review by imay — APPROVED** [review]

**Pull request merged**